### DISCUSSION TITLE
How to use date picket into an extension?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#11026] [Archive]
How to use date picket into an extension?

**Comment by rijkvanzanten** [comment]
All global components are available for use in extensions. You can use `<v-date-picker />` in your extension 👍🏻

**Reply by biapar** [reply]
I already use in that manner but I don't se the input field but only the calendar.

**Reply by rijkvanzanten** [reply]
Ah. That's just a menu with an input:

```vue
<v-menu>
  <template #activator="{ toggle, active }">
    <v-input :active="active" clickable readonly @click="toggle" />
  </template>

  <v-date-picker />
</v-menu>
```

Alternatively, you could re-use the existing interface:

```vue
<interface-datetime />
```
